ciao-cli/instance.go

```diff
@@ -519,9 +519,14 @@ func (cmd *instanceAddCommand) run(args []string) error {
 		fatalf(err.Error())
 	}
 
+	if len(servers.Servers) < cmd.instances {
```

Comment:
On reflection, this is not a new problem.  LaunchInstances returns an error without this patch if any of the instances fail to launch.  I'll therefore enter a separate bug for this.